### PR TITLE
added presentations & publications sections to main and mlir site

### PR DESCRIPTION
**Pull request by martin-luecke**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -128,6 +128,87 @@ <h2>
...
+						<span style="font-family: 'Roboto', sans-serif;"><span
+							style="font-weight: 300;">R</span><span style="font-weight: 400;">I</span><span
+							style="font-weight: 500;">S</span><span style="font-weight: 700;">E</span></span> in MLIR - A functional pattern-based dialect 
+						<br>
+						Martin Lücke, Michel Steuwer
+						<br>
+						<a href="https://www.c4ml.org/c4ml2020"><i class="fab fa-2x" aria-hidden="true"></i>
+							2nd C4ML workshop, at CGO 2020</a>	
+						<br>
+						<a href="http://rise-lang.org/presentations/2020/accml_RISE_in_MLIR_Bologna_2020.pdf" style="margin-right: 1em;">
```
Comment (michel-steuwer) [inline index.html]: Change the link to the poster first.

File: mlir.html [hunk 2]
```diff
@@ -43,19 +43,103 @@ <h2>
...
+						<span style="font-family: 'Roboto', sans-serif;"><span
+							style="font-weight: 300;">R</span><span style="font-weight: 400;">I</span><span
+							style="font-weight: 500;">S</span><span style="font-weight: 700;">E</span></span> in MLIR - A functional pattern-based dialect 
+						<br>
+						Martin Lücke, Michel Steuwer
+						<br>
+						<a href="https://www.c4ml.org/c4ml2020"><i class="fab fa-2x" aria-hidden="true"></i>
+							2nd C4ML workshop, at CGO 2020</a>	
+						<br>
+						<a href="http://rise-lang.org/presentations/2020/accml_RISE_in_MLIR_Bologna_2020.pdf" style="margin-right: 1em;">
```
Comment (michel-steuwer) [inline mlir.html]: Same here: update the poster link.

**Review by michel-steuwer — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (martin-luecke) [reply index.html]: Fixed in https://github.com/rise-lang/rise-lang.github.io/pull/4/commits/761105828a4e58d105b14475a3457ce0703896aa

File: mlir.html (hunk 2 above)
Comment (martin-luecke) [reply mlir.html]: same here: https://github.com/rise-lang/rise-lang.github.io/pull/4/commits/761105828a4e58d105b14475a3457ce0703896aa

**Review by michel-steuwer — APPROVED** [review]

**Pull request merged**